draft-bbs-signatures.md

```diff
@@ -255,34 +258,67 @@ Aside from the message generators, the scheme uses two additional generators: `H
 
 ## Encoding of elements to be hashed
```

Comment:
Should we shift this as an operation under section 5 (https://identity.foundation/bbs-signature/draft-bbs-signatures.html#name-utility-operations)?

Reply:
Done!

Probably best since we are referencing `hash_to_scalar` also!